client/Combatant/StaticCombatantViewModel.ts

```diff
@@ -17,6 +18,7 @@ module ImprovedInitiative {
             this.Initiative = combatant.Initiative();
             this.IsPlayerCharacter = combatant.IsPlayerCharacter;
             this.Tags = combatant.Tags();
+            this.AllowSuggestions = Store.Load(Store.User, "PlayerViewAllowPlayerSuggestions");
```

Comment:
I initially added the flag on a per-combatant because I thought in the future, it would be useful to toggle suggestions on and off on a per-combatant basis.  For example, allowing suggestions only for player characters, or manually allowing suggestions on a certain combatant.  I can change it, but I'd like to hear your thoughts on that.